src/vulnpy/trigger/__init__.py

```diff
@@ -0,0 +1,30 @@
+TRIGGER_MAP = {
```

Comment:
do you think we could dynamically assemble the trigger map by searching for `do_*` functions? It would be really clean if we only had to define a trigger function and then everything else was handled, in my opinion

Reply:
I think we can! addressing the other things you suggested so I may do it now or I may create a ticket for it